src/components/Autosuggest/Autosuggest.js

```diff
@@ -3,7 +3,10 @@ import PropTypes from 'prop-types';
 import Downshift from 'downshift';
 import memoize from 'fast-memoize';
 import bem from '../../utils/bem';
-import { List, ListItem, ContentPlaceholder, Text, MarkedText, FieldWrapper } from '../../index';
```

Comment:
why cannot we import everything together? seems to be easier to type and easier to refactor